### ISSUE TITLE
🎨 Children PIDs no longer displayed

### ISSUE THREAD

**Issue by DevDevious** [gofiber/fiber#843]
Hello,

I noticed that versions 2+ no longer display children process PIDs in the startup message. Would it be possible to revert to previous behavior ?

**Comment by Fenny** [comment]
Hi @DevDevious, I would accept a PR implement the child PIDs in the new startup message.
```
 ┌───────────────────────────────────────────────────┐
 │                    Fiber v2.0.4                   │
 │               http://127.0.0.1:3000               │
 │                                                   │
 │ Handlers ............. 0  Threads ............ 16 │
 │ Prefork ........ Enabled  PID ............. 21220 │
 └───────────────────────────────────────────────────┘
```

**Comment by DevDevious** [comment]
@Fenny Alright, will give it a go when time allows.

**Comment by Fenny** [comment]
I can help out with the implementation, providing a sketch on how we could present it is a good start 🆗

**Comment by DevDevious** [comment]
@Fenny could you please clarify who is supposed to provide the sketch? Is it me or you? 😕

**Comment by Fenny** [comment]
If you could provide the sketch that would be great 👍

**Comment by mahtuog** [comment]
@Fenny , would like to contribute if possible. Haven't used fiber before , but I could see that the pids are obtained in prefork.go when config.prefork is set to true. The v1 branch has both pid and child pids in last fprintf command . v2 branch however, has only pid. We would need to add the pids here, while maintaining the formatting.

**Comment by codemicro** [comment]
Perhaps something like this would be good, where PID is the parent process?
```
 ┌───────────────────────────────────────────────────┐
 │                    Fiber v2.0.4                   │
 │               http://127.0.0.1:3000               │
 │                                                   │
 │ Handlers ............. 0  Threads ............ 16 │
 │ Prefork ........ Enabled  PID ............. 21220 │
 | Child PIDs ... 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, │
 | 12, 13, 14, 15, 16                                |
 └───────────────────────────────────────────────────┘
```

**Comment by mahtuog** [comment]
What would be the appropriate value for Child PIDs field if prefork is disabled ?

**Comment by codemicro** [comment]
I think the best option would be to omit it completely.

**Comment by DevDevious** [comment]
> I think the best option would be to omit that line completely.

That or `N/A` (not applicable)

**Comment by Fenny** [comment]
Or have a separate 
```
 ┌───────────────────────────────────────────────────┐
 │                    Fiber v2.0.4                   │
 │               http://127.0.0.1:3000               │
 │                                                   │
 │ Handlers ............. 0  Threads ............ 16 │
 │ Prefork ........ Enabled  PID ............. 21220 │
 | Child PIDs ... 21220, 21220, 21220, 21220, 21220, |
 | 21220, 21220, 21220, 21220, 21220, 21220, 21220,  |
 | 21220, 21220, 21220, 21220,                       |
 └───────────────────────────────────────────────────┘
```

```
 ┌───────────────────────────────────────────────────┐
 │                    Fiber v2.0.4                   │
 │               http://127.0.0.1:3000               │
 │                                                   │
 │ Handlers ............. 0  Threads ............ 16 │
 │ Prefork ........ Enabled  PID ............. 21220 │
 └───────────────────────────────────────────────────┘
 ┌───────────────────────────────────────────────────┐
 │ Child PIDs ... 21220, 21220, 21220, 21220, 21220, |
 | 21220, 21220, 21220, 21220, 21220, 21220, 21220,  |
 | 21220, 21220, 21220, 21220, 21220, 21220          |
 └───────────────────────────────────────────────────┘
```


```
 ┌───────────────────────────────────────────────────┐ ┌───────────────────────────────────────────────────┐
 │                    Fiber v2.0.4                   │ │ Child PIDs ... 21220, 21220, 21220, 21220, 21220, |
 │               http://127.0.0.1:3000               │ | 21220, 21220, 21220, 21220, 21220, 21220, 21220,  |
 │                                                   │ | 21220, 21220, 21220, 21220, 21220, 21220          |
 │ Handlers ............. 0  Threads ............ 16 │ └───────────────────────────────────────────────────┘
 │ Prefork ........ Enabled  PID ............. 21220 │
 └───────────────────────────────────────────────────┘
```

**Comment by codemicro** [comment]
The third option is the best looking, in my opinion.

**Comment by codemicro** [comment]
Has been implemented with #900.
